tutorials/zero-to-deploy-with-chef.md

```diff
@@ -0,0 +1,328 @@
+          credential 'mycred'
+        end
+
+        gcompute_disk 'instance-test-os-1' do
+          action :create
+          source_image 'projects/ubuntu-os-cloud/global/images/family/ubuntu-1604-lts'
+          zone 'us-west1-a'
+          project ENV['GCP_PROJECT']
+          credential 'mycred'
+        end
+
+        gcompute_network 'mynetwork-test' do
+          action :create
+          project ENV['GCP_PROJECT']
+          credential 'mycred'
+        end
+
+        gcompute_region 'us-west1' do
+          action :create
+          project ENV['GCP_PROJECT']
+          credential 'mycred'
+        end
+
+        gcompute_address 'instance-test-ip' do
+          action :create
+          region 'us-west1'
+          project ENV['GCP_PROJECT']
+          credential 'mycred'
+        end
+
+        gcompute_machine_type 'n1-standard-1' do
+          action :create
+          zone 'us-west1-a'
+          project ENV['GCP_PROJECT']
+          credential 'mycred'
+        end
+
+        gcompute_instance 'instance-test' do
+          action :create
+          machine_type 'n1-standard-1'
+          disks [
+            {
+              boot: true,
+              auto_delete: true,
+              source: 'instance-test-os-1'
+            }
+          ]
+          network_interfaces [
+            {
+              network: 'mynetwork-test',
+              access_configs: [
+                {
+                  name: 'External NAT',
+                  nat_ip: 'instance-test-ip',
+                  type: 'ONE_TO_ONE_NAT'
+                }
+              ]
+            }
+          ]
+          zone 'us-west1-a'
+          project ENV['GCP_PROJECT']
+          credential 'mycred'
+        end
+
+    NOTE: This example code is pulled from the [Google Compute Engine Chef
+    Cookbook](https://github.com/GoogleCloudPlatform/chef-google-compute).
+
+1.  Set the appropriate environment variables. You can directly inline these
+    values in the code; they are parameterized like this for your convenience.
+
+        # The service account key JSON file you uploaded earlier to
+        # '~/credentials.json'. However, CRED_PATH requires an absolute path.
+        export CRED_PATH='/home/USERNAME/credentials.json'
+        export GCP_PROJECT='YOUR_PROJECT_NAME'
+
+    NOTE: Feel free to experiment with more example code from any of the other
+    GCP cookbooks. (e.g. the [Google Cloud SQL Chef
+    Cookbook](https://github.com/GoogleCloudPlatform/chef-google-sql#example)).
+
+## Deploy
+
+### Run Chef Client
+
+On `chef-workstation`, run `chef-client` in 'local mode' with your recipe:
+
+    chef-client --local-mode --runlist 'recipe[google-cloud::default]'
+
+You should see output streaming by as the command operates. It should terminate
+with something like `Chef Client finished, 2/8 resources updated in 35 seconds`.
+
+Awesomesauce! You just provisioned a Compute Engine instance on GCP using a
```

Comment:
Avoid local expression when writing for an international audience.
Suggestion: Change Awesomesauce to Awesome.